src/react/src/login/Login.jsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { useNavigate } from 'react-router-dom';
+import Alert from '@mui/material/Alert';
+import { styled } from '@mui/material/styles';
+import Paper from '@mui/material/Paper';
+import Store from '../redux/store';
+
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  textAlign: 'center',
+  color: theme.palette.text.secondary,
+}));
+function Login() {
+  const navigate = useNavigate();
+  const [errorMessage, setErrorMessage] = React.useState(<p />);
+  const [formValues, setFormValues] = React.useState({
+    username: {
+      value: '',
+    },
+    password: {
+      value: '',
+    },
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    const formFields = Object.keys(formValues);
+    let newFormValues = { ...formValues };
+    const index = formFields.indexOf(name);
+    const currentField = formFields[index];
+    newFormValues = {
+      ...newFormValues,
+      [currentField]: {
+        ...newFormValues[currentField],
+        value,
+      },
+    };
+    setFormValues(newFormValues);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (formValues.username.value !== '' && formValues.password.value !== '') {
+      const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(Object.fromEntries(new FormData(event.currentTarget).entries())),
+      };
+      fetch('/webapp/api/auth/login', requestOptions)
+        .then((response) => response.text())
+        .then((token) => {
+          Store.dispatch({ type: 'ADD_TOKEN', payload: token });
+          if (token.length !== 0) {
```

Comment:
Bitte Login.css in den login-Ordner schieben und den unnötigen Ordner login_component löschen.